tests/Components/Validation/DoesNotContainUrlRuleTest.php

```diff
@@ -10,6 +10,7 @@ class DoesNotContainUrlRuleTest extends TestCase
     const STRING_WITH_URL = 'This is a string http://example.com with an url in it.';
     const STRING_WITHOUT_URL = 'This is a string without an url in it.';
     const STRING_WITH_A_REGRESSION = 'JSON not possible with MSSQL running on Windows platform?';
```

Comment:
Can you remove this constant again? I forgot to remove it in a previous commit :')

Reply:
Done!